README.md

```diff
@@ -78,6 +78,9 @@ You'd have in your `app/views/foo/index.html.haml`:
 
 ```haml
 = table_for @foos [header_visible: (true|false), body_visible: (true|false), footer_visible: (true|false), downloadable: (:xlsx|:csv)] do
+  - row do |attributes,foo|
```

Comment:
Can we consider also the case that we want to add an static class to the whole row by doing it with an `{}` parameter? like
```ruby
- row class: 'foo' do |attributes, foo|
  attributes['data-foo-id'] = foo.id
```

wdyt?